stories/AutocompleteAsync.stories.tsx

```diff
@@ -0,0 +1,251 @@
+          summary: "string",
+        },
+      },
+      control: { type: 'text' }
+    },
+    error: {
+      description: 'Does autocomplete box contain error?',
+      table: {
+        defaultValue: { summary: 'false' },
+        type: {
+          summary: "boolean",
+        },
+      },
+      control: { type: 'boolean' }
+    },
+    errorText: {
+      type: { name: 'string' },
+      description: "Text to display if error",
+      table: {
+        defaultValue: { summary: 'null' },
+        type: {
+          summary: "string",
+        },
+      },
+      control: { type: 'text' }
+    },
+    onChange: {
+      type: { name: 'function' },
+      description: "The onChange handler for change events. Returns the clicked option, to handle as you please.",
+      table: {
+        type: {
+          summary: "function",
+        },
+      },
+    },
+    autoComplete: {
+      description: 'Set built in autocomplete on and off',
+      options: ['off', 'on'],
+      table: {
+        defaultValue: { summary: 'off' },
+        type: {
+          summary: 'string',
+        },
+      },
+      control: { type: 'inline-radio' },
+    },
+    displayValueAndText: {
+      description: 'If true then displays both the value and the text',
+      table: {
+        defaultValue: { summary: 'false' },
+        type: {
+          summary: "boolean",
+        },
+      },
+      control: { type: 'boolean' }
+    },
+    required: {
+      description: 'Sets the input field to required and adds an asterisk',
+      table: {
+        defaultValue: { summary: 'false' },
+        type: {
+          summary: "boolean",
+        },
+      },
+      control: { type: 'boolean' }
+    },
+    prefixIcon: {
+      description: 'Sets an icon at the beginning of the input field',
+      table: {
+        defaultValue: { summary: 'false' },
+        type: {
+          summary: "React.ReactNode",
+        },
+      },
+      control: { type: 'object' }
+    },
+    placeholder: {
+      description: 'Text that is shown when no value is chosen or the dropdown is open',
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+      control: { type: 'text' }
+    }
+  }
+};
+
+const Template = args => {
+  const [_, updateArgs] = useArgs();
+
+  const handleSelect = (option) => {
+    console.log("handleSelect");
+    const newValue = args.value === option?.value ? '' : option?.value;
+    updateArgs({ ...args, value: newValue });
+  }
+  const handleChange = (option: MdAutocompleteAsyncOptionProps) => {
+    const newValue = args.value === option?.value ? '' : option?.value;
+    updateArgs({ ...args, value: newValue });
+  };
+
+  const optionsLoader = async (input: string): Promise<MdAutocompleteAsyncOptionProps[]> => {
+    console.log("optionsLoader");
+    const filteredOptions = [
+      { value: 'optionA', text: 'Eple' },
+      { value: 'optionB', text: 'Banan' },
+      { value: 'optionC', text: 'Appelsin' },
```

Comment:
Eventuelt bruke de samme options-verdiene som i autocomplete for konsistens skyld? Ikke viktig, da :D